requirements/edx/base.txt

```diff
@@ -150,6 +150,7 @@ django_debug_toolbar==1.5
 sqlparse>=0.2.0,<0.3.0
 
 # Used for testing
+astroid==1.3.8
```

Comment:
This seems like a reasonable compromise.  Can we put this next to the edx-lint requirement, with a comment?